src/helpers/webhook-proxy.ts

```diff
@@ -22,8 +22,8 @@ export const createWebhookProxy = async (
 
 export interface WebhookProxyOptions {
   url: string;
-  port?: number;
-  path?: string;
+  port: number;
+  path: string;
   logger: Logger;
-  fetch?: Function;
+  fetch?: typeof fetch;
```

Comment:
This was tagged as `Function` in order to allow different fetch functions that may not all have the same types, but would be functionally compatible.

Ex: if a user would want to use `node-fetch`, would it comlpain?

Reply:
```suggestion
  fetch?: Function;
```